aries-site/src/layouts/navigation/Search.js

```diff
@@ -65,9 +65,9 @@ export const Search = ({ focused, setFocused }) => {
 
   const onSelect = event => {
     const href = nameToPath(event.suggestion);
-
     if (internalLink.test(href)) {
       router.push(href);
+      setFocused(false);
```

Comment:
Do we need to add this here? If the site navigates will setFocused already change to false?

Reply:
yes I added it because with keyboard usage without setting `focus(false)` the input was staying open after the user selected a  suggestion and clicked enter.

Reply:
Great, thanks for clarifying.